packages/scrollable-image-ui/webpack.config.js

```diff
@@ -11,6 +12,20 @@ const config = {
   output: {
     path: path.resolve(__dirname, 'dist/'),
     filename: '[name].[hash].bundle.js',
+    // For path translation, use url provided by unpkg rather than relative path.
+    //
+    // For example:
+    //   This is the url of cloud function we deploy: `https://cloud-function-host/scrollable-image`
+    //   The translated url: `https://lab.twreporter.org/projects/scrollable-image`
+    //
+    //   In this case, index page will be located on `https://lab.twreporter.org/project/scrollable-image/index.html`.
```

Comment:
The URL we want to make public is 
`https://lab.twreporter.org/projects/scrollable-image`,
and we want to proxy pass it to
`https://cloud-function-host/scrollable-image/index.html`.

Therefore, in users' browser, the URL would be
`https://lab.twreporter.org/projects/scrollable-image`.

If we use relative path to be the url of static assets, it would be like
`https://lab.twreporter.org/projects/some-script.js`, and that would be proxy passed to `https://cloud-function-host/projects/some-script.js`.

And it is incorrect URL.

Reply:
Updated.